src/malli/time.clj

```diff
@@ -0,0 +1,69 @@
+(ns malli.time
+  (:require
+   [clojure.string :as str]
+   [malli.core :as m])
+  (:import
+   (java.time Duration LocalDate OffsetDateTime OffsetTime)))
+
+(defn ->safe-parser
+  [f]
+  (fn -parse [x]
+    (if (string? x)
+      (try
+        (f x)
+        (catch Exception _ x))
```

Comment:
Shall we try `Throwable` to catch really everything and make `(f x)` totally safe?

Reply:
Isn't the `->safe-parser` silently swallowing data replacing it with `nil` when it can't parse it?